### PR TITLE
Add command wrapper functionality.

### PR DESCRIPTION
**Pull request by hashbrowncipher**
Goals:
 1) provide a single deployable with no dependencies other than Python +
    Python stdlib.
 2) the deployable should be able to execute a Nagios plugin.
 3) the deployable should combine the output of the Nagios plugin and
    provided Sensu check information.

This implements the above goals by building them into the pysensu_yelp
__init__.py. The bad news here is that it mixes the concern of (a)
reporting results into Sensu and (b) executing Nagios-compatible
plugins. The good news is that it is the simplest way to achieve the
goals above, which I believe to be worthy.

### CODE REVIEW

**Comment by hashbrowncipher** [comment]
Example invocation: `./sensu_reporter.py '{ "ticket": true, "source": "smartstack", "team": "noop", "name": "yo", "runbook": "y/rb-yo" }' -- /bin/bash -c 'exit 2'`

**Comment by solarkennedy** [comment]
Seems fine, but shouldn't this be in its own file so you can add it as a `console_scripts` ?

**Comment by hashbrowncipher** [comment]
Yeah, this is the controversial bit. My proposal is _not_ to use Python packaging, and instead to deploy the entire functionality as a single-file script. In other words, a user without `pip` or `dh-virtualenv` should be able to use the command wrapper functionality by just deploying this single file with their config-management tool.

**Comment by solarkennedy** [comment]
Oh, now I understand what you mean by that.

WFM, shipit

File: pysensu_yelp/__init__.py [hunk 1]
```diff
@@ -210,3 +212,28 @@ def send_event(
...
+                         stdout=subprocess.PIPE,
+                         stderr=subprocess.STDOUT)
+    output, _ = p.communicate()
+    status = p.wait()
+
+    if status > 3:
+        status = 3
+
+    sensu_dict['status'] = status
+    sensu_dict['output'] = output[:1200]
```
Comment (solarkennedy) [inline pysensu_yelp/__init__.py]: 1200 "sounds" small to me. Sensu doesn't limit it, any particular reason why we need to limit it too much here?
https://github.com/sensu/sensu/blob/6df74023f74f67077583e96088a1ecf8ad0651de/lib/sensu/client/process.rb#L151

File: pysensu_yelp/__init__.py [hunk 2]
```diff
@@ -210,3 +212,28 @@ def send_event(
...
+    sensu_dict = json.loads(args.sensu_dict)
+
+    p = subprocess.Popen(args.command,
+                         stdout=subprocess.PIPE,
+                         stderr=subprocess.STDOUT)
+    output, _ = p.communicate()
+    status = p.wait()
+
+    if status > 3:
+        status = 3
```
Comment (solarkennedy) [inline pysensu_yelp/__init__.py]: Can you use the UNKNOWN enum?

**Pull request merged**